### ISSUE TITLE
Using expressions within script tags

### ISSUE THREAD

**Issue by danhalliday** [reshape/expressions#26]
Is it possible to use `{{}}` expressions within script tags? My use case is JSON+LD:

```javascript
<script type="application/ld+json">
  {
    "@context": "http://schema.org",
    "@type": "WebSite",
    "description": "{{ site.description }}"
  }
</script>
```

**Comment by jescalan** [comment]
Unfortunately it isn't - this is a limitation of `parse5` - script tags are "text-only". Eventually I would like to fix this, but unfortunately I don't have the bandwidth right now 😢

**Comment by danhalliday** [comment]
No problem, totally understand! I guess this is the same problem that hits with trying to use expressions within the `title` tag which seems to be parsed as text as per the spec.

**Comment by jescalan** [comment]
Exactly. I'm interesting in building an alternative parser that is much looser in its requirements and faster, but I am not going to have space to work on this for a while, unfortunately. Getting htmlparser2 to work is also on my list to look into. Initially, I didn't use it because it doesn't include location information, which means error messages are brutal - however, that might have changed at this point. If you're interested in working on this, it is definitely a cool project, would be happy to help however I can!
